common/scala/src/main/scala/whisk/core/containerpool/logging/DockerToActivationLogStore.scala

```diff
@@ -67,16 +67,17 @@ class DockerLogStore(system: ActorSystem) extends LogStore {
   override def fetchLogs(activation: WhiskActivation): Future[ActivationLogs] = Future.successful(activation.logs)
 
   override def collectLogs(transid: TransactionId,
+                           user: Identity,
+                           activation: WhiskActivation,
                            container: Container,
                            action: ExecutableWhiskAction): Future[ActivationLogs] = {
 
-    val possibleErrors = Set(Messages.logFailure, Messages.truncateLogs(action.limits.logs.asMegaBytes))
-
     container
       .logs(action.limits.logs.asMegaBytes, action.exec.sentinelledLogs)(transid)
-      .via(DockerLogStore.toFormattedString)
+      .via(DockerToActivationLogStore.toFormattedString)
       .runWith(Sink.seq)
       .flatMap { seq =>
+        val possibleErrors = Set(Messages.logFailure, Messages.truncateLogs(action.limits.logs.asMegaBytes))
```

Comment:
should this be a singleton set?

Reply:
Can't be, the truncation message is specific to the limit.